rust/azure_iot_operations_services/src/leased_lock.rs

```diff
@@ -73,7 +67,7 @@ pub enum ErrorKind {
     /// The payload of the response does not match the expected type for the request.
     #[error("Unexpected response payload for the request type: {0}")]
     UnexpectedPayload(String),
-    /// A lock may only have one [`LockObservation`] at a time.
+    /// A lock may only have one [`LeaseObservation`] at a time.
```

Comment:
is this for lease or lock? You've got some mismatched wording

Reply:
For lease. Fixed.